teos/src/main.rs

```diff
@@ -182,7 +182,7 @@ async fn main() {
     );
     let mut derefed = bitcoin_cli.deref();
     // Load last known block from DB if found. Poll it from Bitcoind otherwise.
-    let last_known_block = dbm.lock().unwrap().load_last_known_block();
```

Comment:
Why would you need to do this :joy: ?
Are you running on mainnet?

Reply:
Yup, I don't follow either

Reply:
my bad,  i used it for testing, forgot to reverse the changes :smile: